qemu/tests/ceph_image_mem_leak.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1) Start guest with 10 rbd data disks and system disk.
+    2) Check the block info for 2 hours
+    3) Check the used memory size is not increased
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    # delay 2 minutes to wait the guest boot up completely
+    time.sleep(120)
+
+    pid = vm.process.get_pid()
+    qemu_stat_file = "/proc/" + str(pid) + "/status"
+    used_mem_cmd = "cat %s | grep VmRSS | awk '{print $2}'" % qemu_stat_file
```

Comment:
VmRSS is using kb currently from the output, but I am not sure if the unit could change, e.g. kb -> mb, then we will get the wrong numbers with different units (kb, mb)

So what about getting the number and its unit together and then use utils_numeric.normalize_data_size() to convert it to kb, e.g. normalize_data_size(size, order_magnitude='K'), then no matter the unit is k or m, we use 'k'.

Reply:
also, you may use a function, e.g. _get_qemu_vmrss() to encapsulate it

Reply:
Updated, thanks. And I changed all the unit to bytes, I believe it would be better.